src/renderer/src/views/components/AiTab/index.less

```
@@ -1371,19 +1371,49 @@
+  }
+
+  .context-truncated-content {
+    display: inline-flex;
+    align-items: center;
+    gap: 6px;
+    white-space: nowrap;
+  }
 
   .context-truncated-icon {
     font-size: 14px;
+    color: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)